tests/utils/test_proto_json_utils.py

```diff
@@ -104,36 +188,19 @@ def test_parse_tf_serving_input():
     with pytest.raises(
         MlflowException, match="The length of values for each input/column name are not the same"
     ):
-        parse_tf_serving_input(tfserving_input)
-
-    # values for each column are properly converted to a tensor
-    arr = [
-        [[1, 2, 3], [4, 5, 6], [7, 8, 9]],
-        [[3, 2, 1], [6, 5, 4], [9, 8, 7]],
-    ]
-    tfserving_input = {"instances": arr}
-    result = parse_tf_serving_input(tfserving_input)
```

Comment:
why was this test case removed?

Reply:
Umm, not removed, it was moved here : https://github.com/mlflow/mlflow/pull/4099/files/c1544c874d7be8dcbf3b2afabe349db8e251fba6#diff-5bf4b1bc0b3499b52dd643e44054f4e7410d5e549fb7673ce0c45757f74ff11cR142

Reply:
oh i'm just bad at reading oops